### PR TITLE
docs: Updated README.md contents in the installing Jenkins plug-in section

### PR DESCRIPTION
**Pull request by ritz303**
Added three additional steps in the Install RHDA Jenkins plug-in section, if the user's build pipeline is not finding the `mvn` binary. Plus did a few other minor edits.

Jira: [TC-530](https://issues.redhat.com/browse/TC-530)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,28 +20,35 @@
...
-- Seach for `redhat-dependency-analytics` and install.
+### 2. Install the redhat-dependency-analytics Jenkins plugin
+- From the Jenkins Dashboard, click **Manage Jenkins** -> **Plugins** -> **Available Plugins**.
+- If Maven is not installed, search for _Maven Integration_, and install the plug-in.
+- Search for _redhat-dependency-analytics_, and install the plug-in.
+- Restart Jenkins.
+
+  **NOTE:** If you get a Java runtime error because the build can not find the `mvn` binary, try doing the following steps:
+  1. From the Jenkins Dashboard, click **Manage Jenkins** -> **System** -> **Check Environment Variables**, click **Add**.
+  2. Enter _EXHORT_MVN_PATH_ as the name, with the value pointing to the `mvn` binary from the Maven Integration installation.
```
Comment (mrrajan) [inline README.md:47]: Shall we add the maven path here as <jenkins_home>/tools/hudson.tasks.Maven_MavenInstallation/MAVEN/bin/mvn

File: README.md [hunk 2]
```diff
@@ -50,7 +57,7 @@ The plugin can be used in jenkins as a pipeline task or as a build step.
 ![](./images/credentialsScreen.png)
 
 ### 4. Configuration
```
Comment (olavtar) [inline README.md:59]: As part of the Configuration if they have a snyk token, they should set it up as Environment Variable as well: EXHORT_SNYK_TOKEN

File: README.md (hunk 1 above)
Comment (olavtar) [reply README.md:47]: @mrrajan Yes

Comment (ritz303) [reply README.md]: @mrrajan : Thanks for the feedback!  I've added the Maven path.

File: README.md (hunk 2 above)
Comment (ritz303) [reply README.md:54]: @olavtar : Thanks for the feedback! I've added steps for adding a Snyk token.

**Review by olavtar — APPROVED** [review]
LGTM

**Pull request merged**